genericjpa/db/src/main/java/org/hibernate/search/db/util/impl/EntityManagerWrapper.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.db.util.impl;
+
+/**
+ * Created by Martin on 11.11.2015.
+ */
+public interface EntityManagerWrapper {
```

Comment:
These are needed so that I can plug in the generic version seamlessly. Also I didn't want to implement the full JPA contract when using a Hibernate Session (Hibernate Search only uses these internal afaik) when there is no JPA available.